resources/views/list-view/sorting.blade.php

```diff
@@ -0,0 +1,36 @@
+{{-- list-view.sorting.blade
+
+Renders the dropdown for the sorting select
+To customize it you should shange the UI component used, YOU SHOULD NOT CUSTOMIZE ANYHING HERE
+UI components used:
+  - form/select --}}
+<div class="leading-tight">
+  {{-- Sorting dropdown --}}
+  @component('laravel-views::components.drop-down')
```

Comment:
You can use blade components like `<x-lv-drop-down />` instead, I used this`@components` way since I was supporting Laravel 6